### PR TITLE
Update crashlytics to use cocoapods

### PR DESCRIPTION
**Pull request by SergioEstevao**
Update project to use Fabric and Crashlytics using Cocoapods.

In the process upgrade Crashlytics and Fabric to the latest version and sort the issue of compiling the project using bitcode.

Needs Review: @astralbodies

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -29,7 +29,8 @@ target 'WordPress', :exclusive => true do
   pod 'RxSwift', '~> 2.1.0'
   pod 'SVProgressHUD', '~>1.1.3'
   pod 'UIDeviceIdentifier', '~> 0.1'
-
+  pod 'Fabric'
```
Comment (astralbodies) [inline Podfile]: I'm going to suggest we not include the `Fabric` pod as its a dependency of `Crashlytics` anyway.

**Comment by astralbodies** [comment]
If removing Fabric from the Podfile works then we can :shipit: @SergioEstevao

@sendhil maybe we can add `pod update crashlytics` to the release branch process?

**Comment by SergioEstevao** [comment]
@astralbodies I followed you suggestion and took out Fabric direct dependency on the Pod and it still works ok.

**Comment by sendhil** [comment]
@astralbodies we can run the update for each release.

**Pull request merged**